pre-commit.sh

```diff
@@ -5,14 +5,23 @@ exec 1>&2
 # No tests to update gh-pages
 BRANCH=$(git symbolic-ref --short HEAD 2>/dev/null)
 if [ "$BRANCH" = "gh-pages" ] ||
+   [ "$BRANCH" = "gh-issues" ] ||
    [ -e .git/MERGE_HEAD ]; then
      exit 0
 fi
 
-git stash save -k -q
+STASHED=0
+if test `git status --porcelain | grep '^[A-Z]' | wc -l` -gt 0; then
```

Comment:
Here's a different approach that might work:

list changed drafts
copy affected drafts to temporary names
make "${affected_drafts}.txt"
rm copies